### PR TITLE
Message options

### PR DESCRIPTION
**Pull request by Endercheif**
### Changes

-   `adds`: docstring on `convert_message`
-   `improvements`: a list of items can be used instead in the convert message function.

example usage:

```py
# before
@command
async def help(ctx):
    return Message(content="Help Command", attachements=[img_A, img_b], embed=[an_embed], component=[button1, button2]

# now
@command
async def help(ctx):
    return "Help Command", img_A, img_b, an_embed, button1, button2

```

Both methods work but the new one "converts" it into a message object with all the properties.

 ### Check off the following

-   [ ] I have tested my changes with the current requirements
-   [x] My Code follows the pep8 code style.

### CODE REVIEW

**Comment by Lunarmagpie** [comment]
How should this work with `Channel.send()`, `Message.edit()`, `Ctx.send()`, and `Ctx.update()` (this one is only on my components PR though). I feel like those should be able to leverage this too.

File: pincer/utils/convert_message.py
Comment (trag1c) [inline pincer/utils/convert_message.py:59]:
```suggestion
            else Message(
                client.received_message,
                flags=InteractionFlags.EPHEMERAL
            )
```

**Review by trag1c — APPROVED** [review]

**Comment by Endercheif** [comment]
> How should this work with `Channel.send()`, `Message.edit()`, `Ctx.send()`, and `Ctx.update()` (this one is only on my components PR though). I feel like those should be able to leverage this too.

They could either accept keyword arguments or a list/tuple of arguments. If it's keyword arguments, they would be passed into the message constructor otherwise it would be passed into the `convert_message` function.

**Review by Lunarmagpie — APPROVED** [review]

**Comment by Endercheif** [comment]
I'll wait until #321 is merged then I will update this to work with with `Channel.send()`, `Message.edit()`, `Ctx.send()`, and `Ctx.update()`

**Review by Sigmanificient — APPROVED** [review]

File: pincer/utils/convert_message.py
Comment (Sigmanificient) [inline pincer/utils/convert_message.py:45]: omg how i didn't spotted that, that should be a defaultdict

Comment (Sigmanificient) [inline pincer/utils/convert_message.py:45]: omg how i didn't spotted that, that should be a defaultdict

**Pull request merged**
